src/helpers.ts

```diff
@@ -57,9 +58,19 @@ export function createList(options: { include?: string[], exclude?: string[] } =
     countries = countries.filter((name) => !exclude.includes(name));
   }
 
+  let selectLabel = { label: 'Please select', selected: true };
+  let getSelectItems = () => countries.map((name) => ({ value: name, displayValue: name }));
+  let getLegacySelectItems = () => countries.map((name) => ({ value: name, label: name }));
+
+  let test = getSelectItems.unshift(selectLabel)
```

Comment:
Right, so you'd trying to call an _Array_ function (`unshift`) directly on a function, rather than on any _outputted_ array from this function.

As this file exports functions to be used elsewhere, we'll need to extend the existing functions to allow this option